src/main/scala/uk/gov/nationalarchives/tdr/api/service/AntivirusMetadataService.scala

```diff
@@ -22,15 +22,22 @@ class AntivirusMetadataService(antivirusMetadataRepository: AntivirusMetadataRep
       input.result,
       Timestamp.from(Instant.ofEpochMilli(input.datetime)))
 
-    antivirusMetadataRepository.addAntivirusMetadata(inputRow).map(row => {
-      AntivirusMetadata(
-        row.fileid,
-        row.software,
-        row.softwareversion,
-        row.databaseversion,
-        row.result,
-        row.datetime.getTime
-      )
-    })
+    antivirusMetadataRepository.addAntivirusMetadata(inputRow).map(rowToAntivirusMetadata)
+  }
+
+  private def rowToAntivirusMetadata(row: AvmetadataRow) = {
```

Comment:
Missing explicit return type for function `rowToAntiVirusMetadata`: `: AntivirusMetadata`